<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-OT8OU31" ignored="0" tests="1" failures="1" timestamp="07 Nov 2018 09:16:00 GMT" time="0.1" errors="0">
  <testcase name="test1" time="0.031" classname="Appium">
    <failure type="java.lang.Error" message="Unresolved compilation problems: 
	Syntax error, insert &amp;quot;)&amp;quot; to complete MethodInvocation
	Syntax error, insert &amp;quot;;&amp;quot; to complete Statement
	By cannot be resolved to a variable
	The method name(String) is undefined for the type By
	The method findElement(By) in the type WebDriver is not applicable for the arguments (ISelector)
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
Syntax error, insert ")" to complete MethodInvocation
Syntax error, insert ";" to complete Statement
By cannot be resolved to a variable
The method name(String) is undefined for the type By
The method findElement(By) in the type WebDriver is not applicable for the arguments (ISelector)

at Appium.test1(Appium.java:57)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- test1 -->
</testsuite>
